src/main/java/Main.java

```diff
@@ -1,39 +1,46 @@
 import animals.Animals;
-import animals.Birds;
-import animals.Fish;
-import animals.Mammals;
-
+import animals.birds.Pigeon;
+import animals.birds.Owls;
+import animals.birds.Parrots;
+import animals.fish.Goldenfish;
+import animals.fish.Sharks;
+import animals.fish.Tunas;
+import animals.mammals.Bears;
+import animals.mammals.Elephants;
+import animals.mammals.Lions;
 import java.util.ArrayList;
 import java.util.InputMismatchException;
 import java.util.Scanner;
 //test
 
-public class Zoo {
+public class Main {
     public static void main(String[] args) {
```

Comment:
The method is HUGE. Would probably make sense to split it into more portions.

Maybe move every switch (menu level) to a new method.